### PR TITLE
[Doc enhancement][App manifest schema]

### PR DESCRIPTION
**Pull request by v-habibaumme**
Added a note

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/170ee7a8a09bcd6ca9081027ff87a53f7abd04b1/msteams-platform/resources/schema/manifest-schema.md) | 86 | 91 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1de1b5d3-0d1a-4176-bc6b-e04eabd173ae) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b164b60bc47e603bd02febdb490d7baeda7b45aa/msteams-platform/resources/schema/manifest-schema.md) | 86 | 91 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1cf2073b-263f-4642-ab99-f35ab288ab2f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b164b60bc47e603bd02febdb490d7baeda7b45aa/msteams-platform/resources/schema/manifest-schema.md) | 86 | 91 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/827530ee-156e-4245-a061-39fca91ae397) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/schema/manifest-schema.md [hunk 1]
```diff
@@ -720,6 +720,9 @@ When a group install scope is selected, it will define the default capability wh
 
 The `configurableProperties` block defines the app properties that Teams admins can customize. For more information, see [enable app customization](~/concepts/design/enable-app-customization.md).
 
+> [!NOTE]
```
Comment (surbhigupta12) [inline msteams-platform/resources/schema/manifest-schema.md:723]: Please combine both notes

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/db5d2d395d0ecb6bfacb105ea2b7672d97e02c63/msteams-platform/resources/schema/manifest-schema.md) | 86 | 91 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/41b481af-f0b0-49d9-a8a2-30a3bad73b86) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by vganiga — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5f3a056111d767adcec459fd151c20ec648586a6/msteams-platform/resources/schema/manifest-schema.md) | 86 | 91 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/05371b25-bb93-431c-b10d-3ce581c3577f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6c001bb58eb73da81a8ccd447e81c074a8539f48/msteams-platform/resources/schema/manifest-schema.md) | 86 | 91 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9f9107f2-8899-4987-9126-141ac19e2088) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/design/enable-app-customization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b2d95172b46382b40e44aef2d3fa487207513cd5/msteams-platform/concepts/design/enable-app-customization.md) | 90 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ac4fb6a3-774c-4a3f-ab3c-986f833bef44) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b2d95172b46382b40e44aef2d3fa487207513cd5/msteams-platform/resources/schema/manifest-schema.md) | 86 | 91 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7010944b-4623-4647-99ff-6365b4e4c293) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/design/enable-app-customization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/847e817ceb304eba67447e70aceda657f2653959/msteams-platform/concepts/design/enable-app-customization.md) | 90 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c7fca537-13fb-4d75-9ada-f40d20d91db2) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/847e817ceb304eba67447e70aceda657f2653959/msteams-platform/resources/schema/manifest-schema.md) | 86 | 91 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/de878a24-b74e-4066-bd77-129d523afb6f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/design/enable-app-customization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5c426c0016e2419690fcf2d519a7b44c1a2b565e/msteams-platform/concepts/design/enable-app-customization.md) | 90 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f36e99c1-c454-4292-99bf-ef4dfd2af89d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/5c426c0016e2419690fcf2d519a7b44c1a2b565e/msteams-platform/resources/schema/manifest-schema.md) | 86 | 91 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/ee75b04c-75e3-4f00-90e9-0e594ca7352d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/design/enable-app-customization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d540982169edb32e4758c489ba5328b82afe318a/msteams-platform/concepts/design/enable-app-customization.md) | 90 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/229aba45-7dfc-43a9-8776-bf06e196cb69) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d540982169edb32e4758c489ba5328b82afe318a/msteams-platform/resources/schema/manifest-schema.md) | 86 | 91 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/002a296b-1194-4443-b58b-795f7a483885) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/design/enable-app-customization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b6dd0d8383cc5cf95fff738aa3eaee72f6381ed3/msteams-platform/concepts/design/enable-app-customization.md) | 90 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/be7ec7d5-fde5-4f0f-90b3-29adc65cb706) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b6dd0d8383cc5cf95fff738aa3eaee72f6381ed3/msteams-platform/resources/schema/manifest-schema.md) | 86 | 91 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e8e7062a-ffac-477d-8abc-e2a658f5fe19) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/design/enable-app-customization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b900ec9e4e72793fac11677d943694bb45469be3/msteams-platform/concepts/design/enable-app-customization.md) | 90 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2025305a-a0de-4c0f-b6ae-b1ffacda852e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/b900ec9e4e72793fac11677d943694bb45469be3/msteams-platform/resources/schema/manifest-schema.md) | 86 | 91 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/31018896-872b-436f-8921-da365cd8ebb3) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by vganiga — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]
Approved

**Pull request merged**